src/rust/ide/view/src/documentation.rs

```diff
@@ -222,36 +259,43 @@ impl View {
 
     /// Constructor.
     pub fn new(scene:&Scene) -> Self {
-        let network = default();
-        let frp     = visualization::instance::Frp::new(&network);
-        let model   = ViewModel::new(scene);
+        let frp               = Frp::new_network();
+        let visualization_frp = visualization::instance::Frp::new(&frp.network);
+        let model             = ViewModel::new(scene);
         model.load_waiting_screen();
-        Self {model,frp,network} . init(scene)
+        Self {model,visualization_frp,frp} . init()
     }
 
-    fn init(self, scene:&Scene) -> Self {
-        let network = &self.network;
-        let model   = &self.model;
-        let frp     = &self.frp;
-        frp::extend! { network
-            eval frp.set_size  ((size) model.set_size(*size));
-            eval frp.send_data ([frp, model](data) {
-                if let Err(e) = model.receive_data(data) {
-                    frp.data_receive_error.emit(Some(e));
+    fn init(self) -> Self {
+        let network       = &self.frp.network;
+        let model         = &self.model;
+        let visualization = &self.visualization_frp;
+        let frp           = &self.frp;
+        frp::extend! { TRACE_ALL network
+
+
+            // === Displaying documentation ===
```

Comment:
I know this is not formal, I will add it to guideline, but lets not use double enter if this is first item in the block - just use single, it looks much muhch better